### PR TITLE
16bit image and mim file

### PR DESCRIPTION
**Pull request by ttagu99**
I need to draw label by 16bit Tiff Image  or mim Image.
mim file is matrox image library file.

### CODE REVIEW

File: labelme/app.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
 from labelme.widgets import LabelQListWidget
 from labelme.widgets import ToolBar
 from labelme.widgets import ZoomWidget
-
+from skimage import io as skio
```
Comment (wkentaro) [inline labelme/app.py]: `scikit-image` is not included in the dependencies. Isn't that possible to use PIL to open mim file?

Comment (ttagu99) [reply labelme/app.py]: PIL can not open the mim file. If skimge can not be added to the dependency, this pull request will be closed.

Comment (wkentaro) [reply labelme/app.py]: It can be an optional dependency.

**Comment by wkentaro** [comment]
Can you give me a sample file of .mim?

**Comment by ttagu99** [comment]
16bit mim file is attatched.

2019년 3월 2일 (토) 오전 2:03, Kentaro Wada <notifications@github.com>님이 작성:

**Comment by ttagu99** [comment]
> Can you give me a sample file of .mim?

[16bit_gray.zip](https://github.com/wkentaro/labelme/files/2925654/16bit_gray.zip)
mim file can not be attached so, I attached zip file

**Comment by wkentaro** [comment]
I a bit worked on this and found there are a lot of image file formats: https://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html,
and still `.mim` file is not listed there.
If I support all of the file extensions, `labelme` needs to load a lot of images to construct the file list, which is not a good thing.

I think maybe you should first convert `.mim` files to a standard file (e.g., `.jpg` or `.png`) and then annotate on them.
What do you think?

**Comment by wkentaro** [comment]
Now I think maybe we can add additional image extensions with the custom configuration.

**Comment by ttagu99** [comment]
> I a bit worked on this and found there are a lot of image file formats: https://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html,
> and still `.mim` file is not listed there.
> If I support all of the file extensions, `labelme` needs to load a lot of images to construct the file list, which is not a good thing.
> (also, maybe in the future someone may say we should support `.npy` file also..)
> 
> I think maybe you should first convert `.mim` files to a standard file (e.g., `.jpg` or `.png`) and then annotate on them.
> What do you think?

I will use forked repo. so this pull request is closing

**Pull request closed without merging**